config.schema.yaml

```diff
@@ -14,3 +14,21 @@
     type: "string"
     secret: false
     required: true
+  sensor:
+    type: "object"
```

Comment:
Need to add `additionalProperties: false` attribute there otherwise user will be able to provider arbitrary invalid options as part of setup key.

For example:

```yaml
setup:
  foo: 'bar'
  bar: 'baz'
```

This would all work if `additionalProperties: false` is not specified.